index.css

```diff
@@ -0,0 +1,540 @@
+  font-family: 'Lato', sans-serif;
+  color: #515151;
+  width: 89%;
+  background: #f8f8f8;
+  font-weight: 400;
+  font-size: 0.75rem;
+  line-height: 1rem;
+  border: 4px solid #fff;
+  padding: 0.5rem;
+  text-align: left;
+  margin-bottom: 2rem;
+}
+
+.date {
+  font-family: 'Gothic A1', sans-serif;
+  font-weight: 700;
+  font-size: 1.1rem;
+  color: #595959;
+  letter-spacing: -0.08em;
+  padding-bottom: 0.2rem;
+}
+
+.location {
+  font-family: 'Gothic A1', sans-serif;
+  font-weight: 600;
+  font-size: 0.7rem;
+  color: #868686;
+  letter-spacing: -0.08em;
+  padding-bottom: 0.2rem;
+}
+
+.program {
+  width: 100%;
+  background-image: url('./images/cross.svg');
+  background-repeat: repeat;
+  background-color: #282b32;
+  mix-blend-mode: multiply;
+}
+
+.program-wrapper {
+  width: 90%;
+  margin: 0 auto;
+  padding: 2rem 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.program-title {
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+  font-size: 1.25rem;
+  line-height: 1.75rem;
+  color: #fff;
+  padding: 0.5rem;
+}
+
+.program-wrapper hr {
+  width: 15%;
+  border: 2px solid #a04038;
+  margin-bottom: 2rem;
+}
+
+.program-card {
+  display: grid;
+  grid-template-columns: 1fr 1fr 2fr;
+  justify-content: center;
+  align-items: center;
+  background: #3b3e45;
+  padding: 1rem;
+  margin-bottom: 0.5rem;
+}
+
+.program-name {
+  font-family: 'Lato', sans-serif;
+  font-weight: 700;
+  font-size: 0.8rem;
+  line-height: 1.5rem;
+  color: #ff5438;
+}
+
+.program-desc {
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+  font-size: 0.8rem;
+  line-height: 1.2rem;
+  letter-spacing: 0.06em;
+  padding: 0.05rem;
+  color: #fff;
+  text-align: left;
+}
+
+.join-btn {
+  font-family: 'Lato', sans-serif;
+  font-weight: 600;
+  font-size: 1rem;
+  color: #fff;
+  padding: 1.25rem;
+  background: #ff4f38;
+  margin-top: 1rem;
+}
+
+.speakers {
+  width: 100%;
+  background: #fff;
+}
+
+.speaker-wrapper {
+  width: 85%;
+  margin: 0 auto;
+  padding: 1.5rem 0;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.speaker-wrapper h2 {
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+  font-size: 1.25rem;
+  line-height: 1.75rem;
+  color: #000;
+  padding: 0.5rem;
+  text-align: center;
+}
+
+.speaker-wrapper hr {
+  width: 45px;
+  border: 1.5px solid #a04038;
+  margin-bottom: 1rem;
+}
+
+.speaker-details {
+  display: grid;
+  grid-template-columns: 40% 60%;
+  margin-bottom: 1rem;
+}
+
+.speaker-img img {
+  width: 50%;
+  aspect-ratio: 1;
+}
+
+.speaker-details .info h3 {
+  font-family: 'Lato', sans-serif;
+  font-weight: 700;
+  font-size: 1rem;
+  line-height: 1.1rem;
+  color: #000;
+  margin-bottom: 0.2rem;
+}
+
+.speaker-details .info .s-title {
+  font-family: 'Lato', sans-serif;
+  font-weight: 700;
+  font-size: 0.6rem;
+  line-height: 1rem;
+  color: #f78c89;
+  margin-bottom: 0.2rem;
+}
+
```

Comment:
- [OPTIONAL] Please consider adding `font-style: italic;` here 🤓🔧I'm sure this will definitely enhance your design ✨